org.gnome.seahorse.Application.json

```diff
@@ -125,8 +115,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/seahorse/3.32/seahorse-3.32.2.tar.xz",
-                    "sha256": "9de021088fc9233087ef7759d9762d86b8162723f403417bdad29d4feb6f1f35"
+                    "url": "https://download.gnome.org/sources/seahorse/3.34/seahorse-3.34.tar.xz",
```

Comment:
this should be 3.34.1

https://download.gnome.org/sources/seahorse/3.34/seahorse-3.34.1.tar.xz and https://download.gnome.org/sources/seahorse/3.34/seahorse-3.34.1.sha256sum